src/app/app-routing.module.ts

```diff
@@ -1,10 +1,16 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import { AboutpageComponent } from './aboutpage/aboutpage.component';
+import { HomepageComponent } from './homepage/homepage.component';
 
-const routes: Routes = [];
+const routes: Routes = [
+  { path: 'aboutpage', component: AboutpageComponent },
+  { path: 'homepage', component: HomepageComponent },
+  { path: '', redirectTo: '/homepage', pathMatch: 'full' },
```

Comment:
`HomePage` should not have a separate route. It should be `''` and any unrecognized route should be redirected to `HomePage`